### ISSUE TITLE
Failed to get access to local media. Black screen.

### ISSUE THREAD

**Issue by aktywnitu** [priologic/easyrtc#145]
I use the newses browser and unfortunatelly have problem with easyrtc  (trunk version).
In Video Only demo I have connected first in Chrome - succesfully, and later in Firefox with error:

Error messages
Failed to get access to local media. Error code was Starting video failed.

In Console:
"Initializing." demo_video_only.js:40
"getusermedia failed" easyrtc.js:2075
"invoking error callback" "Starting video failed" easyrtc.js:2090

In the opposite way is no error, after clicking in chat member button I accept invitation, but unfortunatelly i see black screen in chrome and white in Firefox - the second camera view works properly).

I run server locally under 8088 port: http://localhost:8088/

Can you tell me why it does not work?
Do you have some online demo, which I have test like for example:
https://apprtc.appspot.com/
which works perfectly, but use google signal server.

**Comment by EricDavies** [comment]
our online demos are at http://demo.easyrtc.com
I just tried the simple audio video demo with Firefox 38.0.5 and
Chrome 43.0.2357.81.
No problems.
The most likely cause is that you've got something else using the camera
(like Skype). Cameras frequently aren't sharable devices.

Try running the demos after a reboot.

**Comment by aktywnitu** [comment]
Thank for answer.
Easyrtc have really great api, but unfortunatelly still problems.
Yes I have installed, but closed. Even I open skype https://apprtc.appspot.com/ works properly, but are problems with easyrtc.
I test it on one machine, maby on two different it works, but I am going to deveop app in one machine, so I need see the results.
I am unable test my app, so I have decided deveop my app based on apprtc and run own signaling and stun server.

All the best!
